aquia-inc/template-vite-react
# build(deps): update yarn to 4.4.0 and resolve dependency changes

## Summary

Upgraded Yarn to version 4.4.0 and updated associated configuration and dependencies to reflect this change.

### Added

- None

### Changed

- Upgraded Yarn from 4.3.1 to 4.4.0 in `.yarnrc.yml` and `package.json`.
- Updated `yarn.lock` to reflect changes in dependencies due to the Yarn upgrade.

### Deprecated

- None

### Removed

- Removed `react-dom@npm:18.2.0` from `yarn.lock` as it was replaced by `react-dom@npm:18.3.1`.
- Removed `scheduler@npm:0.23.0` from `yarn.lock` as it was replaced by `scheduler@npm:0.23.2`.

### Fixed

- None

## How to test

1. Checkout this branch: `git checkout origin/build/upgrade-yarn`
2. Run `yarn` and ensure all dependencies are correctly installed without duplication and that there are no issues related to missing or incompatible dependencies.
3. Run `yarn test` and verify that all tests run and pass.
4. Run `yarn dev` and verify that the application runs as expected.
5. Run `yarn build` and verify that the application builds as expected.
6. Run `yarn build-storybook` and verify the Storybook build succeeds.
